client/hijack.go

```diff
@@ -100,7 +91,7 @@ func (cli *Client) setupHijackConn(ctx context.Context, req *http.Request, proto
 			return nil, "", err
 		}
 		if resp.StatusCode != http.StatusSwitchingProtocols {
-			resp.Body.Close()
+			_ = resp.Body.Close()
```

Comment:
It feels like if we're going to adopt the explicit discarded return paradigm, we should consider re-enabling the relevant linters, so we don't "regress" in the affected files again...